packaging/functions.sh

```diff
@@ -62,6 +62,12 @@ install_assemblies() {
 		echo "Installing engine to ${DEST_PATH}"
 		install -d "${DEST_PATH}"
 
+		printf '#!/bin/sh\nexec mono --debug "$0.dll" "$@"\n' >"${DEST_PATH}/OpenRA.Server"
```

Comment:
There are several things here that i'm not a fan of:
* This seems to be removing the `{DEBUG}` placeholder but leaves the `sed` lines that replace its value as dead code.
* Why `exec`?
* This is writing a file for the server, but not for the game?
* This belongs in `install_linux_shortcuts`.
* I don't think we gain much from this additional indirection.

It would still be cleaner and simpler for `install_linux_shortcuts` to set the correct launch statement directly when generating the launcher files.

Reply:
> I don't think we gain anything from the indirection of calling through an additional script.

https://github.com/OpenRA/OpenRA/pull/20054#discussion_r887256949
> I like this idea because the check is done once at install time instead of unnecessarily every launch.

Please make up your mind.

Reply:
Ok, keep the script but please fix the other points.